templates/template_data.go

```diff
@@ -172,69 +172,80 @@ func extendAlert(alert template.Alert, externalURL string, logger log.Logger) *E
 		Fingerprint:  alert.Fingerprint,
 	}
 
+	if generatorURL, err := url.Parse(extended.GeneratorURL); err != nil {
+		level.Debug(logger).Log("msg", "failed to parse generator URL while extending template data", "url", extended.GeneratorURL, "error", err.Error())
```

Comment:
Let's make it warn? I think this is important enough problem

Reply:
Not sure this one is important enough to Warn on but I'll change it as it seems correct in context. The other one for `baseURL` should definitely be Warn though.